packages/contracts-bedrock/test/kontrol/scripts/run-kontrol.sh

```diff
@@ -36,9 +37,15 @@ kontrol_prove() {
     $break_on_calls \
     $break_every_step \
     $tests \
-    --init-node-from $state_diff \
-    --kore-rpc-command 'kore-rpc-booster --equation-max-recursion 100' \
-    --xml-test-report
+    --init-node-from-diff $state_diff \
```

Comment:
Are there docs that explain these options?

Reply:
There are [these docs](https://github.com/runtimeverification/kontrol/tree/master/docs/external-computation), explaining the previous version. Initially, the only option was `from-diff`, but we now support the `dumpState` cheatcode, so we need to differentiate between the `diff` and the `dump`. Documentation on how to get the state updates from the `dumpState` cheatcode can be found [in this tutorial](https://github.com/runtimeverification/kontrol-workshop-ethcc-7/tree/main/4-external-computation), although I haven't managed to update everything together yet